src/wallet/spend.cpp

```diff
@@ -321,7 +321,7 @@ CoinsResult AvailableCoins(const CWallet& wallet,
     std::set<uint256> trusted_parents;
     for (const auto& entry : wallet.mapWallet)
     {
-        const uint256& wtxid = entry.first;
+        const uint256& txid = entry.first;
```

Comment:
(Followup) should check length

Reply:
Not sure what you mean here? `entry.first` is a `uint256`.

Reply:
I think I copy-pasted this comment in the wrong file. Can be ignored